src/crypto/index.js

```diff
@@ -769,6 +769,15 @@ Crypto.prototype.encryptEvent = function(event, room) {
  *  is a problem decrypting the event.
  */
 Crypto.prototype.decryptEvent = function(event) {
+    if (event.isRedacted()) {
+        return {
```

Comment:
This is supposed to return a promise; suggest you wrap it with `Promise.resolve`